pages/registration.page.js

```diff
@@ -8,13 +8,18 @@ let registrationFormNicknameLocator = by.name('login');
 let registrationButtonLocator = by.css('input[type="submit"]');
 let registrationTextLocator = by.css('.h3');
 
+let registrationFormErrorTextLocator = by.css('div.errors');
+//  let registrationFormPasswordErrorTextLocator = by.css('.errors');
+//  let registrationFormNicknameErrorTextLocator = by.css('.errors');
+
 class RegistrationPage extends BasePage {
     async waitForRegisterToBeClickable() {
         await this.getRegistrationButtonElement().waitForElementToBeClickable(100000);
     }
 
+
     // action methods
-    async registration(email, password, nickname) {
+    async registration(email, password, nickname) { //  SUCCESSFULL registration scenario
         await allure.createStep(`Step 3 - proceed with registration '${email}' '${nickname}'`, async () => {
```

Comment:
No need step numbers, just general info. Step number could be different

Reply:
done